### PR TITLE
firewall: add dmz and guests zone

### PR DESCRIPTION
**Pull request by gsanchietti**
See https://community.nethserver.org/t/nethsecurity-zones/23024

See also https://github.com/NethServer/nethsecurity-ui/pull/204

### CODE REVIEW

File: firewall.rst [hunk 1]
```diff
@@ -43,6 +43,47 @@ In this configuration, the firewall regulates traffic between the WAN and LAN zo
 
 Default zones can't be deleted but the network administrator can change existing policies or create new zones. 
 
+Guests and DMZ zones
+--------------------
+
+In addition to the default zones, the firewall can be configured with additional zones to accommodate specific network requirements.
+Two common examples are the Guests and DMZ (Demilitarized Zone) zones.
```
Comment (filippocarletti) [inline firewall.rst:50]:
```suggestion
Two common examples are the Guest and DMZ (Demilitarized Zone) zones.
```

File: firewall.rst [hunk 2]
```diff
@@ -43,6 +43,47 @@ In this configuration, the firewall regulates traffic between the WAN and LAN zo
...
+Sometimes Guests zone is also named as vlue zone while DMZ zone is also named as orange zone.
```
Comment (filippocarletti) [inline firewall.rst:51]:
```suggestion
Sometimes the Guest zone is also known as the blue zone while the DMZ is also named as orange.
```

File: firewall.rst [hunk 3]
```diff
@@ -43,6 +43,47 @@ In this configuration, the firewall regulates traffic between the WAN and LAN zo
...
+
+Guests zone (blue)
+^^^^^^^^^^^^^^^^^^^
+
+Guests zone represents a network segment for guest devices, such as visitors or temporary users.
```
Comment (filippocarletti) [inline firewall.rst:56]:
```suggestion
The Guest zone represents a network segment for guest devices, such as visitors or temporary users.
```

File: firewall.rst [hunk 4]
```diff
@@ -43,6 +43,47 @@ In this configuration, the firewall regulates traffic between the WAN and LAN zo
...
+This zone is typically isolated from the LAN zone to prevent unauthorized access to internal resources.
+But it is allowed to access the WAN zone.
+
+To create a Guests zone, follow these steps:
```
Comment (filippocarletti) [inline firewall.rst:60]:
```suggestion
To create a Guest zone, follow these steps:
```

File: firewall.rst [hunk 5]
```diff
@@ -43,6 +43,47 @@ In this configuration, the firewall regulates traffic between the WAN and LAN zo
...
+
+- access the ``Zones & policies`` section
+- click on the ``Add zone`` button
+- enter **guests** inside the ``Name`` field: plesase note that the name is case sensitive and must be in lowercase, in this case the zone will be highlighted in blue
```
Comment (filippocarletti) [inline firewall.rst:64]: There's been some talks about naming the zone "guest" instead of guests.

File: firewall.rst [hunk 6]
```diff
@@ -43,6 +43,47 @@ In this configuration, the firewall regulates traffic between the WAN and LAN zo
...
+- select ``LAN`` inside the ``Allow forwards from`` field
+- enable the ``Traffic to WAN`` option
+- select ``Drop`` for both ``Traffic to firewall`` and ``Traffic for the same zone`` fields
+- click on the ``Save`` button and apply the changes
+
+
+DMZ zone (orange)
+^^^^^^^^^^^^^^^^^
+
+DMZ zone represents a network segment for servers and services that need to be accessible from the internet but isolated from the internal network.
```
Comment (filippocarletti) [inline firewall.rst:75]:
```suggestion
The DMZ represents a network segment for servers and services that need to be accessible from the internet but isolated from the internal network.
```

File: firewall.rst [hunk 7]
```diff
@@ -43,6 +43,47 @@ In this configuration, the firewall regulates traffic between the WAN and LAN zo
...
+
+To create a DMZ zone, follow these steps:
```
Comment (filippocarletti) [inline firewall.rst:77]:
```suggestion
To create a DMZ, follow these steps:
```

**Review by filippocarletti — CHANGES_REQUESTED** [review]

File: firewall.rst (hunk 5 above)
Comment (gsanchietti) [reply firewall.rst:64]: Where?

**Pull request merged**